### PR TITLE
feat: add DeePMD installation agent skill

### PR DESCRIPTION
**Pull request by OutisLi**
## Summary

- add the `deepmd-install` Agent Skill with concise routes for pip, conda, dp1s, offline packages, Docker, source builds, C/C++, and LAMMPS
- fetch version-matched official documentation before rendering version-sensitive commands, using exact-ref raw Markdown from GitHub or the official Gitee mirror
- treat dynamic command values as literal process arguments rather than shell source
- pin pip and Conda installations to the selected release and fail verification on a normalized version mismatch
- make source runtime and compiled-backend selection explicit, including the documented backend-neutral C/C++ route
- keep a compact failure-mode reference for environment identity, backend and accelerator selection, source builds, native loader checks, Docker, and LAMMPS
- document skill installation in the README and installation docs, using the official Gitee mirror as the network fallback

## Design

The skill provides route selection, safety boundaries, and verification criteria without duplicating the installation manual. It assumes no local source checkout. The agent opens the direct official page first, resolves a release to its tag, and falls back to the corresponding Markdown at that exact tag or commit when versioned web documentation is unavailable.

Source installation remains the recommended path when a compiler is available and build time is acceptable. TensorFlow, PyTorch, JAX, Paddle, and backend-neutral C/C++ libraries are represented, while version-specific package, accelerator, and build commands remain owned by the official documentation.

## Validation

- official `quick_validate.py skills/deepmd-install`
- repository pre-commit hooks for all changed files
- shell-free argv round-trip with spaces, command-substitution syntax, backticks, semicolons, and option-looking values
- PEP 440 release normalization and exact pip requirement parsing
- Conda exact-version MatchSpec parsing, including rejection of the fuzzy single-equals form
- direct GitHub and official Gitee raw-documentation fetch at an exact upstream commit
- official Gitee and upstream `master` commit identity check
- source build environment-assignment shell check
- local Git checkout checks for both a normal ref and an option-looking ref
- `git diff --check`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,9 +212,27 @@ plugins. Applications can therefore open the backend required by a model
...
+> If direct GitHub access fails, use `gh-proxy.com` for a public, read-only
+> clone, then install from the local checkout. Do not send credentials or
+> private repository URLs through the proxy.
+>
+> ```bash
+> git clone --depth 1 \
+>     https://gh-proxy.com/https://github.com/deepmodeling/deepmd-kit.git \
+>     deepmd-kit-skill-source
+> npx -y skills add ./deepmd-kit-skill-source/skills \
+>     --skill deepmd-install -y
```
[the remark this answers is not in the document]
Comment (njzjz) [reply README.md]: https://gitee.com/deepmodeling/deepmd-kit is official and better

**Review by njzjz — APPROVED** [review]
